### ISSUE TITLE
Issue with xTeVe xml

### ISSUE THREAD

**Issue by mkowpak** [xteve-project/xTeVe#356]
I recently got xTeVe setup, but am running into an issue with the xml provided from xTeVe, and am quite stumped. I have a remote XMLTV file set up from my provider. Accessing this by it's url shows me all the channel and programme info. However, in the exported xTeVe XML, I only get the channels, and not the programmes, so I have no guide data. Am I missing something in my setup?

Edit: I should also note, in xTeVe I see the accurate number of channels and programs

**Comment by MCTyler** [comment]
Did you `Filter` and `Map` the EPG in the `Mapping` section?

**Comment by chamby90** [comment]
@mkowpak i have this same issue.  xteve is unable to recognize the episode details and then just assigns the date as the episode number when i feed it into plex.  @MCTyler , yes, i personally map it in the xteve mapping section and then send the xteve XML into plex for what its worth.

**Comment by cwawak** [comment]
Hi! Running into this as well. Also mapped.

**Comment by requa3r0** [comment]
@chamby90  Are you trying to import VOD i xTeVe ??

That is not what its for. Its only for the LIVE TV section of the M3U file!
Use filters on the live tv groups, and filter only the channels you need...match everything with EPG data!

All VOD (moveis and shows) is indexed in KODI / EMBY / PLEX media sever library.

Use this https://github.com/erdesigns-eu/M3U2STRM

to translate the M3U file.

When this is translated to folders and *.strm files you can index all VOD into KODI / EMBY / PLEX media sever library. 
This give you control, it tracks what is watched, filters and meta-data etc. 

Then import the Live TV in Emby's live tv section or in KODI using the PVR addon.  IPTV Simple Client

ITs easy ;O)

**Comment by chamby90** [comment]
No, I'm not talking about VOD.  Im talking about live TV when I have a show on the epg will not show the  season and episode number.  It just shows todays date.
